### PR TITLE
Database Source Lint

### PR DESCRIPTION
**Pull request by ulixius9**
### Describe your changes :

Database Source Lint

#
### Type of change :

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/database/athena.py [hunk 1]
```diff
@@ -34,12 +36,12 @@
 logger = ingestion_logger()
 
 
-def _get_column_type(self, type_):
+def _get_column_type(self, type_):  # pylint: disable=too-many-branches
```
Comment (TeddyCr) [inline ingestion/src/metadata/ingestion/source/database/athena.py:39]: I think we should not disable that. To me it means we need to maybe change our deep if/else to a look dict or soemthing similar -- example below.

```python
def bad(a):
    if a == 1: return "A"
    if a == 2: return "B"
    if a == 3: return "D"
    if a == 4: return "G"
    if a == 5: return "M"
    if a == 6: return "X"

    return "Z"

def better(a):
    return _LOOKUP.get(a, "Z")

_LOOKUP = {
    1: "A",
    2: "B",
    3: "D",
    4: "G",
    5: "M",
    6: "X",
}
```

File: ingestion/src/metadata/ingestion/source/database/dbt_source.py [hunk 2]
```diff
@@ -204,21 +208,18 @@ def _parse_data_model_upstream(self, mnode):
         return upstream_nodes
 
     def _parse_data_model_columns(
-        self, model_name: str, mnode: Dict, cnode: Dict
+        self, _: str, mnode: Dict, cnode: Dict
     ) -> List[Column]:
         columns = []
         ccolumns = cnode.get("columns")
```
Comment (TeddyCr) [inline ingestion/src/metadata/ingestion/source/database/dbt_source.py:214]: why is that named `ccolumns`?

Comment (ulixius9) [reply ingestion/src/metadata/ingestion/source/database/dbt_source.py:214]: not sure why it's like that but I guess that's pointing to columns of catalogue node

Comment (ulixius9) [reply ingestion/src/metadata/ingestion/source/database/dbt_source.py:214]: will rename this

**Review by OnkarVO7 — APPROVED** [review]

**Review by nahuelverdugo — APPROVED** [review]

**Pull request merged**